collections/harmonized-landsat-sentinel.yaml

```diff
@@ -0,0 +1,526 @@
+Name: Harmonized Landsat Sentinel
+Description: |
+  [Harmonized Landsat Sentinel](https://lpdaac.usgs.gov/data/get-started-data/collection-overview/missions/harmonized-landsat-sentinel-2-hls-overview/)
+  is a NASA initiative to produce a Virtual Constellation of surface reflectance (SR) data from the Operational Land Manager (OLI) and Multi-Spectral
+  Instrument (MSI) aboard the Landsat 8 and Sentinel-2 remote sensing satellites, respectively. The combined measurement enables global observations
+  of the land every 2-3 days. Input products are Landsat 8-9 Collection 2 L1 and S2-L1C top-of-atmosphere reflectance. Landsat data is available from
+  April 2013 and Sentinel-2 data from November 2015. 
+Documentation: "[here](https://docs.sentinel-hub.com/api/latest/data/hls/)"
+Resolution: 30m
+GeographicalCoverage: Globally
+TemporalAvailability: April 2013 - ongoing
+UpdateFrequency: 2-3 days
+BandInformation: "Information about [bands and data](https://docs.sentinel-hub.com/api/latest/data/hls/#available-bands-and-data)"
+Contact: https://forum.sentinel-hub.com/c/euro-data-cube/25
+Provider: "[Land Processes Distributed Active Archive Center (LP DAAC)](https://lpdaac.usgs.gov/)"
+ManagedBy: "[Sentinel Hub](https://www.sentinel-hub.com/)"
+Tags:
+  - xcube
+  - sentinel hub
+  - raster
+  - systematic
+  - satellite imagery
+  - multi spectral imagery
+  - agriculture
+  - natural resource
+  - disaster response
+  - vegetation monitoring
+  - LULC mapping
+  - LULC change
+  - open data
+  - race challenges
+  - copernicus
+  - sentinel
+  - landsat
+License: |
+    "[License](https://www.usgs.gov/centers/eros/data-citation?qt-science_support_page_related_con=0#qt-science_support_page_related_con)"
```

Comment:
This was not rendered correctly in the homepage/JSON. One solution would be to display `License:` as markdown to show both (S2 and L8) licenses in the homepage. `LicenseUrl:` would then be removed since it's not a required element.

Reply:
Done. Also tried `gulp serve` and seems to be generated fine.